Example/package.json

```diff
@@ -7,7 +7,7 @@
   },
   "dependencies": {
     "react": "~15.3.0",
-    "react-native": "^0.34.0",
+    "react-native": "^0.41.2",
```

Comment:
No I don't see a need for testing on iOS as long as no changes are affecting iOS. But when you bump the version it will of course require testing on iOS.